docs/application/native/tutorials/details/native-removed-history.md

```diff
@@ -191,6 +191,109 @@ The following table provides detailed information regarding removed functions an
+| Network - Wi-Fi | wifi_get_connection_state() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_get_connection_state() |
+| Network - Wi-Fi | wifi_set_device_state_changed_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_set_device_state_changed_cb() |
+| Network - Wi-Fi | wifi_unset_device_state_changed_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_unset_device_state_changed_cb() |
+| Network - Wi-Fi | wifi_set_background_scan_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_set_background_scan_cb() |
+| Network - Wi-Fi | wifi_unset_background_scan_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_unset_background_scan_cb() |
+| Network - Wi-Fi | wifi_set_connection_state_changed_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_set_connection_state_changed_cb() |
+| Network - Wi-Fi | wifi_unset_connection_state_changed_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_unset_connection_state_changed_cb() |
+| Network - Wi-Fi | wifi_set_rssi_level_changed_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_set_rssi_level_changed_cb() |
+| Network - Wi-Fi | wifi_unset_rssi_level_changed_cb() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_unset_rssi_level_changed_cb() |
+| Network - Wi-Fi | wifi_ap_create() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_create() |
+| Network - Wi-Fi | wifi_ap_hidden_create() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_hidden_create() |
+| Network - Wi-Fi | wifi_ap_destroy() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_destroy() |
+| Network - Wi-Fi | wifi_ap_clone() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_clone() |
+| Network - Wi-Fi | wifi_ap_refresh() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_refresh() |
+| Network - Wi-Fi | wifi_ap_get_essid() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_get_essid() |
+| Network - Wi-Fi | wifi_ap_get_bssid() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_get_bssid() |
+| Network - Wi-Fi | wifi_ap_get_rssi() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_get_rssi() |
+| Network - Wi-Fi | wifi_ap_get_frequency() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_get_frequency() |
+| Network - Wi-Fi | wifi_ap_get_max_speed() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_get_max_speed() |
+| Network - Wi-Fi | wifi_ap_is_favorite() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_is_favorite() |
+| Network - Wi-Fi | wifi_ap_is_passpoint() | Mobile, Wearable | 3.0 | 5.5 | Better alternative | wifi_manager_ap_is_passpoint |

```

Comment:
wifi_manager_ap_is_passpoint
Isn't wifi_manager_ap_is_passpoint()?

Reply:
Added () for wifi_manager_ap_is_passpoint.